tests/contrib/sensors/test_sqs_sensor.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import unittest
+from airflow import DAG, configuration
+from airflow.contrib.sensors.aws_sqs_sensor import SQSSensor
+from airflow.utils import timezone
+from mock import patch, call, MagicMock
+from airflow.exceptions import AirflowException
+
+DEFAULT_DATE = timezone.datetime(2017, 1, 1)
+
+
+class TestSQSSensor(unittest.TestCase):
+
+    def setUp(self):
+        configuration.load_test_config()
+
+        args = {
+            'owner': 'airflow',
+            'start_date': DEFAULT_DATE
+        }
+
+        self.dag = DAG('test_dag_id', default_args=args)
+        self.sensor = SQSSensor(
+            task_id='test_task',
+            dag=self.dag,
+            sqs_queue='test',
+            aws_conn_id='aws_default'
+        )
+
+        self.mock_context = MagicMock()
+
+    @patch('airflow.contrib.sensors.aws_sqs_sensor.SQSHook')
```

Comment:
Please can we mock using [moto](http://docs.getmoto.org/en/latest/) instead where possible - we've had problems in the past where mocks are over-specific and don't represent AWS's behaviour.

Reply:
@ashb - I've replaced the success/failure tests with moto.  I've kept the exception based tests for checking those use cases.  Please let me know if you have any additional recommendation.  Thanks